README.md

```diff
@@ -2,20 +2,32 @@
 
 ## EYE1
 
-The legal possession of this `CRED` token is [recordat]
+The legal possession of this `CRED` token is [recordat]() of the _social engagement_, skills, and side-projects of the owner. In conjunction with srCRED [^1] ¿TBA?, it is LexDAO's assessment (passive and active/explicit voting) of participating in legal engineering activities including:
```

Comment:
Cred should be a grantable right by qualified members.  If you have earned the right to vote you can distribute cred of a particular group.  If you have earned a committee role you can distribute cred of a different group.  If you have academic status or certifications etc you have a different group of cred you can distribute.  All at the individual member discretion.  Committees themselves might also offer awards that represent cred of a different order.  I think there is a lot of depth to build out here.